src/graphql/mutations/createEtchPacket.js

```diff
@@ -3,24 +3,28 @@ const defaultResponseQuery = `{
   id
   eid
   name
+  detailsURL
```

Comment:
Let's be mindful of what's added here that may be to satisfy your example repo usage vs. what should truly be sensible defaults. If there's something you think might not reasonably belong in a default response query then let's leave it out...and if your app needs it, then it can pass its own custom response query.

I think that these are all fine in that regard, and no need to change them, but let's be mindful.

Reply:
Agreed and thanks for pointing that out. I'm taking `downloadZipURL` out as I don't see it being necessary.